src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java

```diff
@@ -157,11 +157,15 @@ private static void setChromeDownloadDirectory(ChromeOptions options) {
     
     /**
      * Creates a ChromeDriver. Makes it headless if the -Dheadless flag is set.
+     * Can pass additional arguments with the -DchromeOptions flag, such as -DchromeOptions="start-maximized" to open all browser windows maximized.
      * @return WebDriver ChromeDrvier
      */
     private static WebDriver createChromeDriver() {
     	var chromeOptions = new ChromeOptions();
     	setChromeDownloadDirectory(chromeOptions);
+        String commandlineOptions = Configuration.toString("chromeOptions");
+        if (commandlineOptions != null)
+            chromeOptions.addArguments(commandlineOptions);
```

Comment:
Add a unit test for this. Replace the chrome brad browsertest in WebDriverFactoryTests as that also exists in WebDriverFctoryBadBrowserNameTests.

Reply:
Done!